packages/fluentui/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Fixes
 - `Popup` should not dismiss when its iframe content is focused @ling1726 ([#19955](https://github.com/microsoft/fluentui/pull/19955))
 - `whatInput` Add typings to what input event listeners @ling1726 ([#20024](https://github.com/microsoft/fluentui/pull/20024))
+- Remove `active` as condition to open submenu on `MenuItem` @chassunc ([#20062](https://github.com/microsoft/fluentui/pull/20062))
```

Comment:
also mention the styling change here, it _could_ cause some headaches upgrading

Reply:
Done